torch/_export/__init__.py

```diff
@@ -78,6 +78,36 @@
 from .wrappers import _wrap_submodules
 
 
+class AotInductorTransformationHook:
```

Comment:
Wait sorry, I didn't see this change. I would rather you just pass the passes to export.aot_compile instead of having these hooks.